static/examples/batch/batch-slicing.scd

```diff
@@ -0,0 +1,68 @@
+// Refer to https://learn.flucoma.org/overviews/batch-processing
+(
+s.waitForBoot({
+    fork({
+        var drum = Buffer.read(s,File.realpath(FluidDataSet.class.filenameSymbol).dirname.withTrailingSlash ++ "../AudioFiles/Nicol-LoopE-M.wav");
+        var slices = Buffer(s); // The slicer (FluidBufOnsetSlice) will write into this buffer the samples at which slices are detected
+        var sliceAnalysis = Buffer(s); // A Buffer() for writing the analysis from FluidSpectralShape into
+        var sliceCentroid = Buffer(s); // A Buffer() for extracting the centroid into
+        var sliceStats = Buffer(s, 7); // A Buffer() for writing the statistical analyses of the centroid into
+        var sliceMean = Buffer(s, 1); // A Buffer() for extracting just the mean from the 7 statistical analyses.
+
+        var data = FluidDataSet(s); // A FluidDataSet() for storing the analysis of each slice (mean centroid)
+        s.sync;
```

Comment:
Don't think this is needed because of the `wait` below

Reply:
Seems to be okay without it.